tests/suites/test_suite_psa_crypto_entropy.function

```diff
@@ -2,17 +2,38 @@
 #include <stdint.h>
 
 #include "psa/crypto.h"
-#include "psa_prot_internal_storage.h"
 #include "mbedtls/entropy.h"
 #include "mbedtls/entropy_poll.h"
 
+#if defined(MBEDTLS_PSA_ITS_FILE_C)
+#include <stdio.h>
+#else
+#include <psa/internal_trusted_storage.h>
+#endif
+
 /* Calculating the minimum allowed entropy size in bytes */
 #define MBEDTLS_PSA_INJECT_ENTROPY_MIN_SIZE MAX(MBEDTLS_ENTROPY_MIN_PLATFORM, MBEDTLS_ENTROPY_BLOCK_SIZE)
 
+/* Remove the entropy seed file. Since the library does not expose a way
+ * to do this (it would be a security risk if such a function was ever
+ * accessible in production), implement this functionality in a white-box
+ * manner. */
+psa_status_t remove_seed_file( void )
+{
+#if defined(MBEDTLS_PSA_ITS_FILE_C)
+    if( remove( "00000000ffffff52.psa_its" ) == 0 )
+        return( PSA_SUCCESS );
+    else
+        return( PSA_ERROR_DOES_NOT_EXIST );
+#else
+    return( psa_its_remove( PSA_CRYPTO_ITS_RANDOM_SEED_UID ) );
```

Comment:
Couldn't `psa_its_remove()` work whether `MBEDTLS_PSA_ITS_FILE_C` is defined or not? We have it implemented already since we need `psa_its_remove()` for removing keys, although this implementation here looks a bit simpler (no `fopen()`, single call to `remove()`).

Reply:
I guess it could, but there'd need to be more code here to include the proper header that declares `psa_its_remove`, or an explicit declaration of `psa_its_remove`. I'd rather not touch this code more than necessary until we've finished properly integrating it and we can test it.